yocto-build-env/s6-overlay/scripts/addusers

```diff
@@ -13,7 +13,8 @@ fetch_ssh_keys() {
         mkdir -p .ssh
         curl -fsSL "https://github.com/${_username}.keys" >> .ssh/authorized_keys
         chown -R "${_username}:${_username}" .ssh
-        chmod -R 600 .ssh
+        chmod -R 700 .ssh
+        chmod -R 644 .ssh/authorized_keys
```

Comment:
Shouldn't this be `600` not `644`?

Reply:
based on [this](https://superuser.com/questions/215504/permissions-on-private-key-in-ssh-folder), `644` for public keys is correct.